packages/annotorious-openseadragon/vite.config.js

```diff
@@ -28,15 +24,18 @@ export default defineConfig({
     },
     rollupOptions: {
       external: [
-        ...Object.keys(packageJson.peerDependencies)
+        ...Object.keys(packageJson.peerDependencies),
+        '@annotorious/core',
+        '@annotorious/annotorious'
```

Comment:
I'm using the explicit exclusion list definition, because `/@annotorious\/(core|annotorious)/` leads to the following warning on bundling:
<img width="823" alt="image" src="https://github.com/annotorious/annotorious/assets/68850090/2f5585f5-f9dc-40bb-bd70-b6d527cb5861">

Reply:
But wouldn't that mean core and annotorious will not get bundled into the UMD bundle? That would IMO lead back to the broken script imports I had a while back.

Reply:
It's worth mentioning that the `annotorious/annotorious` has already been excluded from the bundle before the https://github.com/annotorious/annotorious/pull/386